core/src/main/resources/lib/layout/symbol.jelly

```diff
@@ -1,7 +1,8 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:d="jelly:define">
     <st:documentation>
-        Provides ionicon icons
+        Deprecated
```

Comment:
Code search shows this is only used in core we can just remove it after updating all usages
https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Ajenkinsci+l%3Aionicon